server/gokbg3/grails-app/init/gokbg3/BootStrap.groovy

```diff
@@ -203,6 +203,13 @@ class BootStrap {
       }
       log.debug("${ctr} components updated");
 
+    log.info("GoKB remove usused refdata");
+      def rr_std = RefdataCategory.lookup('ReviewRequest.StdDesc', 'RR Standard Desc 1')
+
+      if (rr_std) {
+        rr_std.delete()
```

Comment:
Cleaning known junk at startup seems weird. Where does this RefdataCategory come from in the first place? Why it's written to the database at all and why isn't that avoided?

Reply:
The value was added via BootStrap.groovy before, which was only removed in a previous patch. We can therefore assume it was at some point created in all existing instances so has to be removed everywhere.